archive_repo.py

```diff
@@ -656,11 +633,14 @@ def log(*str, **kwargs):
             number = pr["number"]
 
             # Since we can't filter, check if we already have this one.
-            if number in pr_ref:
-                ref_updatedAt = dp.parse(pr_ref[number]["updatedAt"])
-                dl_updatedAt = dp.parse(pr["updatedAt"])
-                if ref_updatedAt >= dl_updatedAt:
-                    continue
+            if not updateOld:
+                if number in pr_ref:
```

Comment:
Don't you need to declare `global pr_ref` in order to get access to global state?

Reply:
Technically, no; `global` declares that I'm going to be reassigning a global variable within this scope instead of creating a new variable with the same name.  Accessing globals and interacting with the objects they point at doesn't require a declaration.  Once pr_ref is assigned, it's never reassigned; everything is mutations of the object.